RecoTauTag/HLTProducers/interface/CaloTowerFromL1TSeededCreatorForTauHLT.h

```diff
@@ -0,0 +1,56 @@
+#ifndef CaloTowerCreator_CaloTowerFromL1TSeededCreatorForTauHLT_h
+#define CaloTowerCreator_CaloTowerFromL1TSeededCreatorForTauHLT_h
+
+/** \class CaloTowerFromL1TSeededCreatorForTauHLT
+ *
+ * Framework module that produces a collection
+ * of calo towers in the region of interest for Tau HLT reconstruction,
+ * defined around L1 seeds
+ *
+ * \author T. Strebler. IC.   based on A. Nikitenko
+ *
+ */
+
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DataFormats/CaloTowers/interface/CaloTower.h"
+#include "DataFormats/L1Trigger/interface/Tau.h"
+#include "DataFormats/HLTReco/interface/TriggerFilterObjectWithRefs.h"
+#include <string>
+
+namespace edm {
+  class ParameterSet;
+}
+
+class CaloTowerFromL1TSeededCreatorForTauHLT : public edm::global::EDProducer<> {
+ public:
+  /// constructor from parameter set
+  CaloTowerFromL1TSeededCreatorForTauHLT( const edm::ParameterSet & );
+  /// destructor
+  ~CaloTowerFromL1TSeededCreatorForTauHLT();
+  /// 
+  static void fillDescriptions( edm::ConfigurationDescriptions& desc );
+
+ private:
+  /// process one event
+  void produce( edm::StreamID sid, edm::Event& evt, const edm::EventSetup& stp ) const override;
+
+  /// verbosity
+  const int mVerbose;

```

Comment:
please change the data members to follow the CMSSW naming conventions: either `m_verbose` or `verbose_`, and similarly for the others